cyber-resilience-sig/minutes/deliverables-plan-task-force/2025-01-30-deliverables-plan-tf-minutes.md

```diff
@@ -0,0 +1,55 @@
+# Cyber Resilience SIG - Deliverables Plan Task Force
+
+###  January 30, 2025
+
+In person meeting during the morning and afternoon of the ORC WG Workshop in Brussels.
+
+## Participants
+
+* Tobie Langel (UnlockOpen/Eclipse Foundation)
+* Mikael Barbero (Eclipse Foundation)
+* Timo Perala (Nokia)
+* Jonne Soininen (Nokia)
+* Dirk-Willem van Gulik (ASF)
+* Thierry Carrez (OpenStack)
```

Comment:
He usually lists his affiliation as OpenInfra Foundation (the steward member representing the OpenStack project).